src/coreclr/vm/threads.h

```diff
@@ -4015,14 +4028,26 @@ class Thread
     }
 
 #ifndef DACCESS_COMPILE
-    void     SetSavedRedirectContext(PT_CONTEXT pCtx)
+    void MarkRedirectContextInUse(PTR_CONTEXT pCtx)
```

Comment:
should this also be under `ifdef _DEBUG'?

Reply:
I wanted the actual calls to be not `_DEBUG`, since there are many calls to this and they come in pairs - Set/Unset. Making the calls conditional on `_DEBBUG` looked noisy. 
I opted for just making the validation conditional.